dask_kubernetes/operator/operator.py

```diff
@@ -213,16 +211,17 @@ async def daskworkergroup_update(spec, name, namespace, logger, **kwargs):
                 f"Scaled worker group {name} up to {spec['worker']['replicas']} workers."
             )
         if workers_needed < 0:
-            service_address = await get_scheduler_address(
-                f"{spec['cluster']}-service", namespace
-            )
-            logger.info(
-                f"Asking scheduler to retire {-workers_needed} on {service_address}"
-            )
-            async with rpc(service_address) as scheduler:
-                worker_ids = await scheduler.workers_to_close(
-                    n=-workers_needed, attribute="name"
-                )
+            async with aiohttp.ClientSession() as session:
+                params = {"n": -workers_needed}
+                async with session.post(
+                    "http://localhost:8787/api/v1/retire_workers", json=params
```

Comment:
We can't assume this will be on `localhost`. We still need the `get_scheduler_service` call, but we need to modify that function so we can tell it we want port `8787` rather than `8786`. See my other comment about this.